README.rst

```diff
@@ -37,7 +37,10 @@
 
 A python package implementing the stretched NMF algorithm.
 
-* LONGER DESCRIPTION HERE
+``diffpy.snmf`` is a Python package that increases the insight one can obtain from a measured series time-dependent signals
```

Comment:
Please can you have another go, maybe pulling from the paper or the synopsis or professional summary in the regolith citations database in billingegroup/rh-db-public?